src/modules/sub-items/components/viewing-message/viewing.message.component.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './css/viewing.message.component.css';
+
+const ViewingMessageComponent = ({ totalCount, viewingCount }) => {
+    const message = Translator.trans(
+        /*@Desc("Viewing <strong>%viewingCount%</strong> out of <strong>%totalCount%</strong> sub-items")*/ 'viewing_message',
+        {
+            viewingCount,
+            totalCount,
+        },
+        'sub_items'
+    );
+
+    return (
+        <div className="c-viewing-message">
+            <div className="c-viewing-message__message" dangerouslySetInnerHTML={{ __html: message }} />
```

Comment:
`c-viewing-message__message` looks not so good. Maybe `c-viewing-message__text`?